docs/asyncapi.yaml

```diff
@@ -82,6 +71,20 @@ components:
           $ref: '#/components/schemas/SchemeId'
         value:
           $ref: '#/components/schemas/ValueInteger'
+    PaymentRequestSubmit:
+      type: object
+      required:
+      - paymentRequest
+      - referenceId
+      properties:
+        invoiceLines:
+            $ref: '#/components/schemas/InvoiceLinesSubmit'
+        paymentRequest:
+          $ref: '#/components/schemas/PaymentRequestProcessing'
```

Comment:
Dnt think there's a better way of doing this -- no `{ ...objectSpreaderEquailvalentUnfort, invoiceLines: wouldHaveBeenNice }`

Reply:
Agreed there's not.  Had to do something similar in `ffc-pay-enrichment` for inbound requests from Siti vs non Siti.  Mostly the same but with some additions.